.github/workflows/fuzzers.yml

```diff
@@ -46,9 +46,13 @@ jobs:
     env:
       FUZZER: ${{ matrix.fuzzer }}
       BENCHMARK_TYPE: ${{ matrix.benchmark_type }}
+      ALWAYS_BUILD_FUZZER: afl
```

Comment:
One problem with using src_analyzer to strictly limit what gets built in CI is testing changes to the build system as a whole rather than individual fuzzers or benchmarks.
For example, what should get built if we change docker/build.mk. In theory this affects the build of every fuzzer.
There are two approaches I have taken in this PR, we could drop either one or even combine them.
The first approach (the one I take in this file) is to always build one fuzzer. It's hard for me to imagine a change to build.mk breaking some fuzzers and not others, so maybe building one is sufficient.
This approach saves builds when we do change these files but is otherwise wasteful since we are always building AFL. However, I think the approach might be most likely to catch changes that src_analysis misses and so in some sense it is safest.
The second approach, which I take in change_utils is to build *every* fuzzer when one of these files changes. 
The third approach, which I haven't implemented is to combine the two. Only build one fuzzer when one of these files changes. This is the least conservative approach but saves the most amount of builds.

Any ideas for which I take?

Reply:
If dont take second approach, we will silent failures that will show up later when running experiment. We have to do that, otherwise more manual work and followup stuff. it is resource intensive, but what can we do else.

Reply:
I see what you mean but I think 2 is inherently fragile as we must remember to include every file that affects builds in `CI_FILES`. I think the current implementation which always builds AFL but builds everything when CI_FILES is changed is the best approach. Maybe I'll eliminate the AFL part if I get more confident in the CI_FILES approach.